pkg/validation/internal/operatorhub.go

```diff
@@ -178,6 +180,11 @@ func validateOperatorHub(objs ...interface{}) (results []errors.ManifestResult)
 		}
 	}
 
+	// Add a deprecation warning to the list so that users are aware this validator is deprecated
+	deprecationResultWarning := errors.ManifestResult{}
+	deprecationResultWarning.Add(errors.WarnDeprecatedValidator("OperatorHub Validator deprecated, use OperatorHubV2 validator"))
```

Comment:
Two suggestions:
1. Use the name of the validator as it will show up for a user (see [here](https://github.com/operator-framework/operator-sdk/blob/d21ed6499ebfc8ecdb4508e1c2a2a0cfd2a151f3/internal/cmd/operator-sdk/bundle/validate/optional.go#L40); as an aside, it irks me that the name doesn't live with the code.. are these validators used elsewhere?)
2. Mention the combination of validators that are required to get equivalent validation

Also open to suggestions on how to format the `v2` suffix. Operator SDK/Kubebuilder use `<name>/<version>` for plugin names, so this is probably the most predictable for users.
```suggestion
	deprecationResultWarning.Add(errors.WarnDeprecatedValidator(`The "operatorhub" validator is deprecated; for equivalent validation use "operatorhub/v2" and "standardcategories" validators`))
```

Reply:
I think this mapping is just a convention for the ux in the operatorsdk cli? The only thing this api exposes is the actual public interfaces, which can't have a slash. We'd have to generate these mappings as a wrapper type if we wanted to expose them in a generic way, I think. For now, I think just adding the message in this format seems reasonable though.